src/components/common/Dialogs/CreatePaymentDialog/helpers.ts

```diff
@@ -32,3 +36,33 @@ export const getCreatePaymentDialogPayload = (colony: Colony, payload: any) => {
     motionDomainId,
   };
 };
+
+export const useCreatePaymentDialogStatus = (
+  colony: Colony,
+  requiredRoles: ColonyRole[],
+  enabledExtensionData: EnabledExtensionData,
+) => {
+  const { getValues } = useFormContext();
+  const values = getValues();
```

Comment:
So apparently, after talking to @willm30, I think you should use the `watch` function here. `getValues` doesn't guarantee that the component will get re-rendered if `values.fromDomain` changes
On a side note, you could destructure `fromDomain`